App Submission: DCR Pulse

karamble opened the pull request:
# App Submission

### App name
...

### 256x256 SVG icon
![dcrpulse](https://github.com/user-attachments/assets/02a2920c-b4a1-4978-87b6-ae2ead5a32f8)


### Gallery images
<img width="1440" height="900" alt="screenshot_1" src="https://github.com/user-attachments/assets/fcce3046-6176-4e53-add5-e7a8d57ecc3f" />
<img width="1440" height="900" alt="screenshot_2" src="https://github.com/user-attachments/assets/1371712a-fdd4-4991-aefe-d934cd765f19" />
<img width="1440" height="900" alt="screenshot_3" src="https://github.com/user-attachments/assets/629dd0cc-b698-4ffe-9da6-182414172278" />



### I have tested my app on:
- [X ] umbrelOS on a Raspberry Pi
- [ ] umbrelOS on an Umbrel Home
- [X ] umbrelOS on Linux VM

karamble committed:
App Submission: DCR Pulse

karamble committed:
fix linter errors and app directory

karamble committed:
fix app directory for dashboard volume

karamble committed:
change to user 1000 like in the upstream docker files

al-lac commented on dcrpulse/docker-compose.yml:
```diff
@@ -0,0 +1,100 @@
+    stop_grace_period: 5m
+    dns:
+      - 1.1.1.1
+      - 8.8.8.8
```
Setting the DNS manually here should not be needed.
```suggestion
```

al-lac commented on dcrpulse/docker-compose.yml:
```diff
@@ -0,0 +1,100 @@
+        condition: service_healthy
+    networks:
+      - default
+      - dcrpulse_internal
```
Custom networks should not be defined for umbrel apps.
```suggestion
```

al-lac commented on dcrpulse/umbrel-app.yml:
```diff
@@ -0,0 +1,51 @@
+id: dcrpulse
+name: Decred Pulse
+tagline: Full Decred node with watch-only wallet and database-free block explorer
+category: finance
```
Crypto category would probably fit the best here.
```suggestion
category: crypto
```

al-lac reviewed (changes requested):
Hey @karamble,

thanks a lot for this awesome submission. 🔝
And good job on getting everything in place here. 
Really great addition to the Umbrel Crypto Ecosystem! ⚡ 

I made some suggestions on what should be adjusted to get the app ready for the store.

karamble committed:
changes as per requests on our umbrel PR

- Remove custom networks
- Change port from 8080 to 8735
- Update to dcrd v2.1.2 and dcrwallet v2.1.2
- Update dashboard to v1.0.2
- Change category from finance to crypto
- Fix description formatting for proper line breaks

karamble commented:
all requested changes have been made. Linter error on the requested crypto - category

al-lac reviewed (changes requested):
Hey @karamble,

thanks for your patience on this.

Added some smaller things to take a look at. The app is working just fine as is.

Ignore the crypto category error for now, should actually be fixed with the next action run.

We can already start to prepare the gallery assets for the app.

karamble committed:
Upgrade dcrpulse to v1.0.3 with Decred v2.1.3

jzbz commented:
Hey @al-lac I'm not clear on where the gallery assets need to go to get this merged, can you advise @karamble? Thanks.

al-lac committed:
Update description formatting and add gallery assets

al-lac commented:
Hey @karamble, thanks again for you submission of DCR Pulse to the Umbrel App Store! Also thanks for your patience on this.

I added the gallery assets now, tested it one more time and we should be ready to go live! 🚀 

<img width="2724" height="1780" alt="CleanShot 2026-04-17 at 14 19 27@2x" src="https://github.com/user-attachments/assets/6196f98e-09c3-439b-9661-fb489e5990a7" />
